mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -760,9 +760,9 @@ def get_task_detections(self, num_class_with_bg, batch_cls_preds,
                 selected = nms_gpu(
                     boxes_for_nms,
                     top_scores,
-                    thresh=self.test_cfg['nms_iou_threshold'],
-                    pre_maxsize=self.test_cfg['nms_pre_max_size'],
-                    post_max_size=self.test_cfg['nms_post_max_size'])
+                    thresh=self.test_cfg['nms_thr'],
```

Comment:
Why do we need to rename them here.

Reply:
To make it consistant with the parameter names in other models.